drivers/tm1637/include/tm1637_params.h

```
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2024 Nico Behrens <nifrabe@outlook.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_tm1637
+ *
+ * @{
+ * @file
+ * @brief       Config for the TM1637 display
+ *
+ * @author      Nico Behrens <nifrabe@outlook.de>
+ */
+#ifndef TM1637_PARAMS_H
+#define TM1637_PARAMS_H
+
+#include "board.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef TM1637_PARAM_CLK
+/**
+ * @brief see @ref tm1637_params_t
+ */
+#define TM1637_PARAM_CLK GPIO_PIN(0, 0)
```

Does it make sense to have default values here, or should we rather error out if the user hasn't provided them? Not sure how this is done in other places in RIOT.

---

I would suggest setting the default to `GPIO_UNDEF`. Then add an assert to the init function to check that the pins are defined. There is no sane default you could ever define. Better to fail loudly then quietly do the wrong thing.